database/migrations/2019_06_26_045023_create_admin_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAdminTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('admin', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
- where are the other columns for admin table?
 - please make admin plural

Reply:
admin file was deleted